plugin.video.putio/addon.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+
+<addon id="plugin.video.putio"
+    name="put.io"
+    version="2.0.0"
+    provider-name="Put.io Developers">
+    <requires>
+        <import addon="xbmc.python" version="2.14.0" />
+        <import addon="script.module.requests" version="2.9.1" />
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="main.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.service" library="service.py" start="startup" />
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <summary lang="en">Put.io for Kodi</summary>
+        <description lang="en">Put.io is a service that combines cloud storage and bittorrent. You need an account to use this plug-in. Get one from https://put.io</description>
+        <language></language>
+        <license>GNU GENERAL PUBLIC LICENSE Version 3, 29 June 2007</license>
```

Comment:
please add source field. Pls see http://kodi.wiki/view/Addon.xml for required fields.